### PR TITLE
Add contact form to portfolio

### PR DESCRIPTION
**Pull request by gergitokusse**
### The following file was updated

- index.html file
- style.css file
- readme file was customized
### The following file was added to file
- different background image was added to a img folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,271 +1,511 @@
...
+    <!-- From ensaha -->
+    <section class="form-container-2">
+      <strong class="form-header-2">Contact me</strong>
+      <div class="contact-form-2">
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it!
+        </p>
+        <form class="contact-me-2" action="">
```
Comment (emmiiorji) [inline index.html]: Kindly ensure that the form uses the action and method attributes to integrate with [formspree service](https://formspree.io/html). This is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/91668f1d04ab043c8fac118a333ea1d5695b54b3/portfolio/3_contact_form.md?plain=1#L48) for the project.

In order to let formspree recognize each input field, you may have to specify a name attribute for each of them.

File: style.css [hunk 2]
```diff
@@ -1,645 +1,732 @@
...
+  }
+}
+
+.form-container-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-top-left-radius: 10%;
```
Comment (emmiiorji) [inline style.css]: You seem to be missing out a little detail about the corner radius of the section. Kindly ensure that it's as close to the design as possible.

Also, the background image of the section is missing

Your design:
image

Figma design:
image

File: style.css [hunk 3]
```diff
@@ -1,645 +1,732 @@
...
+    flex-basis: 25rem;
+  }
+}
+
+.form-container-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
```
Comment (emmiiorji) [inline style.css:652]: Kindly also, include this little horizontal bar below the form. Let's get as close to the design as possible.

Your design:
image

Figma design:
image

File: README.md [hunk 4]
```diff
@@ -1,219 +1,219 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gergito K.**
+
+- GitHub: [@githubhandle](https://github.com/gergitokusse)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (emmiiorji) [inline README.md:163]: [Optional] It's a good practice to have the names of the contributors listed. I recommend that you update the list of contributors.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi Team, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>


## Good Highlights✔️

- Good design
- HTML form validation is implemented


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (gergitokusse) [reply index.html]: > Kindly ensure that the form uses the action and method attributes to integrate with [formspree service](https://formspree.io/html). This is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/91668f1d04ab043c8fac118a333ea1d5695b54b3/portfolio/3_contact_form.md?plain=1#L48) for the project.
> 
> In order to let formspree recognize each input field, you may have to specify a name attribute for each of them.

🆗 I am sorry i forgot it i will add wait a moment.

File: style.css (hunk 2 above)
Comment (gergitokusse) [reply style.css]: 🆗 Thank you for your nice coment

File: style.css (hunk 3 above)
Comment (gergitokusse) [reply style.css:652]: 👍 👍 👍  I see it on Figma but understood it as the horizontal scroll of the screen. I will add it 🆗 .

File: README.md (hunk 4 above)
Comment (gergitokusse) [reply README.md:163]: : Wow I am sorry to forget them it also needed to acknowledge them on the acknowledgment part.
👍 👍 👍  Thank you again and again for an interesting comment. 👍 👍 👍

**Review by aamirkhan2478 — APPROVED** [review]
Hi @gergitokusse , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gergitokusse** [comment]
# 🤝 Thanks a lot. 🤝

**Pull request merged**